### DISCUSSION TITLE
Customize placeholder

### DISCUSSION THREAD

**Discussion by scottaw66** [Pagefind/pagefind#107] [Q&A] [answered]
Is there any way to customize the input placeholder text in such a way that it won't get overwritten? I mean the actual placeholder text, not its styling.

Thank you.

**Comment by bglw** [comment, marked as answer]
Created issue #109 and implemented in PR #108 — I'll drop another comment here when I build the next Pagefind release (should be later this week)

**Reply by bglw** [reply]
@scottaw66 This has been implemented in the [Pagefind v0.9.0](https://github.com/CloudCannon/pagefind/releases/tag/v0.9.0) release 🎉

**Reply by scottaw66** [reply]
Thank you so much.
